.github/workflows/python-app.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: gitchangelog
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Git repository creation (required)
+      run: |
+        git init
+        git fetch
+        git reset origin/main
+        git checkout origin/main
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wheel
+        pip install minimock mako pystache
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Do we expect the case when the `if` returns false?

Reply:
if requirements fail, most likely the step `Lint with flake8` will fail.

Reply:
> Do we expect the case when the `if` returns false?

No, not really. If requirements.txt are not installed, tests should currently still pass since we do not (yet) test `kolibree_output`.